app_widgetbook/lib/main.dart

```diff
@@ -452,6 +616,37 @@ class WidgetbookApp extends StatelessWidget {
   }
 }
 
+/// Predefined options knob for a tweet date
+DateTime getTweetDateOption(BuildContext context) {
+  return context.knobs.options<DateTime>(
+    label: 'Date',
+    description: "The date should be a timeago string if it's "
+        'less than 24 hours ago, and it should '
+        "include the year only if it's different "
+        'than the one we are currently in',
+    options: [
+      Option(
+        label: '12 hours ago',
+        value: DateTime.now().subtract(
```

Comment:
I understand that displaying the date like this is not optimal as the `now()` will change after the Widgetbook cloud is deployed and use cases will be messed up (For example, the 12 hour ago would become 13 hours ago if the Widgetbook cloud was open for an hour)

You will also see this not-so-optimal `DateTime.now()` usage in a widget test in this PR.

Usually when I work with DateTime's and I want to test them, I use the [clock](https://pub.dev/packages/clock) package to mock the time and have the `DateTime.now()` and the operations I do on it be fixed for the tests, and in this case also for the Widgetbook components. 

What do you think? Is it okay with you if we use the package or is the current logic not a big issue?